packages/sit-onyx/src/components/OnyxBreadcrumb/OnyxBreadcrumb.stories.ts

```diff
@@ -33,6 +35,14 @@ export const Default = {
     default: () => [
       h(OnyxBreadcrumbItem, { href: "/foo" }, () => "Foo"),
       h(OnyxBreadcrumbItem, { href: "/foo/bar" }, () => "Bar"),
+      h(OnyxBreadcrumbItem, { href: "/foo/bar/baz1" }, () => "Baz1"),
```

Comment:
Should we prefer creating a separate example / story which this many items?
Otherwise the default example looks a little too large imo